generator/cli/generate-type-script.ts

```diff
@@ -144,10 +144,52 @@ const getTemplateInfos = (params: { templatesRepoDir: string }): TemplateInfo[]
   return glob.sync(`${params.templatesRepoDir}/*`, { onlyDirectories: true }).map((path) => ({
     name: templateName(Path.basename(path)),
     displayName: startCase(Path.basename(path)),
+    icon: getTemplateIcon(Path.basename(path)),
+    description: getTemplateDescription(Path.basename(path)),
     path,
   }))
 }
 
+export type iconTypes = 'triangle' | 'music' | 'home' | 'cloud' | 'link-2' | 'battery' | ''
+
+const getTemplateIcon = (id: string): iconTypes => {
+  switch (id) {
+    case 'nextJS':
+      return 'triangle'
+    case 'musicStreamingService':
+      return 'music'
+    case 'rentalsPlatform':
+      return 'home'
+    case 'saas':
+      return 'cloud'
+    case 'urlShortener':
+      return 'link-2'
+    case 'empty':
+      return 'battery'
+    default:
+      return ''
+  }
+}
+
+const getTemplateDescription = (id: string): string => {
```

Comment:
This content would be useful in other static places such as the README and/or as a comment in the PSL.

Let's use that as a source of truth?

Reply:
This can come from the `package.json`'s `description`